ddtrace/contrib/aiopg/connection.py

```diff
@@ -15,21 +15,19 @@ class AIOTracedCursor(wrapt.ObjectProxy):
     def __init__(self, cursor, pin):
         super(AIOTracedCursor, self).__init__(cursor)
         pin.onto(self)
-        name = pin.app or 'sql'
-        self._datadog_name = '%s.query' % name
 
     @asyncio.coroutine
-    def _trace_method(self, method, resource, extra_tags, *args, **kwargs):
+    def _trace_method(self, method, query, extra_tags, *args, **kwargs):
         pin = Pin.get_from(self)
         if not pin or not pin.enabled():
             result = yield from method(*args, **kwargs)  # noqa: E999
             return result
         service = pin.service
 
-        with pin.tracer.trace(self._datadog_name, service=service,
-                              resource=resource) as s:
+        name = (pin.app or 'sql') + "." + method.__name__
```

Comment:
The `operation_name` (span name) must be a static value otherwise it could create problems when the span is a root span for this service. Can't we set that value simply as `sql.query` and then add a tag with this information?

Reply:
this is what it was before, see: https://github.com/DataDog/dd-trace-py/pull/394/files#diff-c2dd489fb57497c257d41a833aeb36faL18  are you saying that was wrong too?